indico/modules/attachments/templates/_attachments.html

```diff
@@ -64,6 +64,14 @@
                    title="{% trans %}The access to this folder is restricted, regardless of the protection scheme of
                           the parent.{% endtrans %}"></i>
             {% endif %}
+            {% if folder.is_hidden and management%}
+                <i class=" icon-eye-blocked"
```

Comment:
there shouldn't be a space right after the opening quote

Reply:
Done